package.json

```diff
@@ -245,10 +245,10 @@
     "tree-kill": "1.2.2",
     "tsec": "0.2.1",
     "turbo": "2.1.2",
-    "typescript": "5.5.3",
+    "typescript": "5.6.2",
     "unfetch": "4.2.0",
     "wait-port": "0.2.2",
-    "webpack": "5.90.0",
+    "webpack": "5.95.0",
```

Comment:
Why do we need to bump this? Let's do this separately if required since the bundler is a very integral part of the codebase.

Reply:
Error with root webpack type:

```
Error: ../../node_modules/.pnpm/webpack@5.90.0_@swc+core@1.7.0-nightly-
20240714.1_@swc+helpers@0.5.13_/node_modules/webpack/types.d.ts(12112,2):
error TS2416: Property '[Symbol.iterator]' in type 'SortableSet<T>' is not assignable
to the same property in base type 'Set<T>'.
```

https://github.com/vercel/next.js/actions/runs/11058324871/job/30724077537#step:8:524

Reply:
reorder stack: bump webpack to 5.95.0 -> bumb ts